po/pl.po

```diff
@@ -367,15 +367,15 @@ msgstr ""
 
 #: src/login/org.freedesktop.login1.policy:180
 msgid "Power off the system while an application is inhibiting this"
-msgstr "Wyłączenie systemu, kiedy program zażądał jego wstrzymania"
+msgstr "Wyłączenie systemu, kiedy program wstrzymuje to"
 
 #: src/login/org.freedesktop.login1.policy:181
 msgid ""
 "Authentication is required to power off the system while an application "
 "is inhibiting this."
 msgstr ""
-"Wymagane jest uwierzytelnienie, aby wyłączyć system, kiedy program zażądał "
-"jego wstrzymania."
+"Wymagane jest uwierzytelnienie, aby wyłączyć system, kiedy program wstrzymuje "
```

Comment:
"Wymagane jest uwierzytelnienie, aby wyłączyć system, kiedy program je wstrzymuje "

Reply:
"to wstrzymuje" ?